clients/src/test/java/org/apache/kafka/common/serialization/SerializationTest.java

```diff
@@ -119,6 +120,49 @@ public void testLongSerializer() {
     }
 
     @Test
+    public void testFloatSerializer() {
+        Float[] floats = new Float[]{
+            5678567.12312f,
+            -5678567.12341f
+        };
+
+        Serializer<Float> serializer = Serdes.Float().serializer();
+        Deserializer<Float> deserializer = Serdes.Float().deserializer();
+
+        for (Float value : floats) {
+            assertEquals("Should get the original double after serialization and deserialization",
+                    value, deserializer.deserialize(topic, serializer.serialize(topic, value)));
+        }
+
+        assertEquals("Should support null in serialization and deserialization",
+                null, deserializer.deserialize(topic, serializer.serialize(topic, null)));
+
+        serializer.close();
```

Comment:
> these calls aren't needed, right? `close` does nothing anyway.

Yes, close doesn't do anything in this specific case, but that behavior is not part of the contract afaict.  Hence IMHO we should always close it.

> And if they did do something, then you'd probably want to do it in `tearDown` rather than here.

That would only work if we created a global serde/serializer/deserializer instance that we'd re-use between tests.  It's a matter of personal I admit, but I'd rather avoid doing that (I'm ok with the re-usable `topic` variable though).  That said, I don't have a strong opinion.  If we do change the pattern, then the existing tests should be updated, too (though outside of this PR) for the sake of consistency.